### PR TITLE
Prevent "undefined" error code

### PR DESCRIPTION
**Pull request by bennyn**

### CODE REVIEW

File: app/script/event/EventRepository.coffee [hunk 1]
```diff
@@ -355,7 +355,7 @@ class z.event.EventRepository
         return @cryptography_repository.decrypt_event event
         .catch (decrypt_error) =>
           # Get error information
-          error_code = decrypt_error.code
+          error_code = decrypt_error.code or 999
```
Comment (gregor) [inline app/script/event/EventRepository.coffee:358]: Can we make it 666 to be the devil?

Comment (lipis) [reply app/script/event/EventRepository.coffee:358]: as long as it's not a magic number.. I'm down

Comment (bennycode) [reply app/script/event/EventRepository.coffee:358]: `666` is the [number of the beast](https://en.wikipedia.org/wiki/Number_of_the_Beast) and I don't want to have Satan in our app. 😋

Comment (lipis) [reply app/script/event/EventRepository.coffee:358]: I wasn't kidding about the "magic" number.. 999 should be a const :|

**Pull request merged**